imcache_test.go

```diff
@@ -941,6 +942,53 @@ func TestImcache_GetAll_SlidingExpiration(t *testing.T) {
 	}
 }
 
+//GetAndRemove Tests Init
+func TestImcache_GetAndRemove(t *testing.T) {
```

Comment:
Please also extend the `TestImcache_Close` test to call the new method, see https://github.com/erni27/imcache/blob/master/imcache_test.go#L1133.